baremaps-cli/src/assembly/filter.properties

```diff
@@ -0,0 +1 @@
+application.version=${project.version}
```

Comment:
Is this file necessary? I don't think it is used elsewhere.

Reply:
![image](https://user-images.githubusercontent.com/10920115/229067169-db2a1b7f-f0da-4d8b-9bfe-8cfa06ca6ce6.png)

It's needed by the `filtered` property of the `fileSet`in the assemblies files.
It's the way to say to the assembly plugin to read a file and inject a property.